src/main/java/com/baran/charactersheetcreator/configuration/SecurityFilterChain.java

```diff
@@ -0,0 +1,31 @@
+package com.baran.charactersheetcreator.configuration;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.core.annotation.Order;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+
+import static org.springframework.security.config.Customizer.withDefaults;
+
+public class SecurityFilterChain {
+    @Bean
+    @Order(1)
```

Comment:
As the meaning of the number is somewhat hidden in the annotation a constant could be created that indicates the precedence of these rules, e.g. HIGHEST_ORDER=9999